### PR TITLE
VZ-10040:update release notes for known issue.

### PR DESCRIPTION
**Pull request by neeraj-n-singh**
We have known issue with fluentd and system compatibility.

### CODE REVIEW

**Review by rosemarymarano — COMMENTED** [review]
Edits needed

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Component version updates:
 - WebLogic Kubernetes Operator v4.0.6
 - WebLogic Monitoring Exporter v2.1.3
 
+Known Issues:
+- The fluentd plug-in used in verrazzano can not read the journal db logs in newer versions of systemd. This specifically impacts on community editions of kindset version 1.24.10 or more and Ubuntu 21.04, Oracle Linux 9 and any operating systems that has that has systemd version 246 or higher.
```
[the remark this answers is not in the document]
Comment (neeraj-n-singh) [reply content/en/docs/releasenotes/_index.md:18]: I will add these changes, also I am waiting for technical feedback from others as well.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Component version updates:
 - WebLogic Monitoring Exporter v2.1.3
 
 Known Issues:
-- The fluentd plug-in used in verrazzano can not read the journal db logs in newer versions of systemd. This specifically impacts on community editions of kindset version 1.24.10 or more and Ubuntu 21.04, Oracle Linux 9 and any operating systems that has that has systemd version 246 or higher.
+- The Fluentd plug-in used in verrazzano cannot read the journal database logs in newer versions of systemd. This specifically impacts on community editions of kindset version 1.24.10 or later and Ubuntu 21.04, Oracle Linux 9 and any operating systems that have systemd version 246 or later.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: -> `journal` (this word should be in code font (surrounded with back ticks))
-> impacts community (remove the word "on")
-> please provide a pointer to "kindset"

**Review by rosemarymarano — COMMENTED** [review]
Few more

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gilbode) [reply content/en/docs/releasenotes/_index.md]: I believe the first sentence should state the symptom.  e.g. Systemd journal logs are not collected on some platforms.  And we should change "fluentd plug-in" to state "fluentd systemd plugin".

[the remark this answers is not in the document]
Comment (neeraj-n-singh) [reply content/en/docs/releasenotes/_index.md]: Added as per feedback. Needs further reviews and feedback.

[the remark this answers is not in the document]
Comment (gilbode) [reply content/en/docs/releasenotes/_index.md]: Suggested wording:

Systemd journal logs are not collected on some platforms.  The [systemd plugin for Fluentd](https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) cannot read systemd journal files on Kubernetes nodes that use newer versions of systemd.  This is known to impact the default node images of [kind](https://kind.sigs.k8s.io/) for Kubernetes 1.24 and later, Oracle Linux 9 and any other Kubernetes cluster nodes running on an operating system that has systemd version 246 or higher.

[the remark this answers is not in the document]
Comment (neeraj-n-singh) [reply content/en/docs/releasenotes/_index.md]: Updated as per suggested.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -14,6 +14,9 @@ Component version updates:
 - WebLogic Kubernetes Operator v4.0.6
 - WebLogic Monitoring Exporter v2.1.3
 
+Known Issues:
+- Systemd journal logs are not collected on some platforms. The [systemd plugin for Fluentd](https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) cannot read systemd journal files on Kubernetes nodes that use newer versions of systemd. This is known to impact the default node images of [kind](https://kind.sigs.k8s.io/) for Kubernetes 1.24 and later, Oracle Linux 9 and any other Kubernetes cluster nodes running on an operating system that has systemd version 246 or higher.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: -> Oracle Linux 9, (please put a comma after this item)

**Review by rosemarymarano — APPROVED** [review]
One minor edit, otherwise fine.

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (neeraj-n-singh) [reply content/en/docs/releasenotes/_index.md:18]: Sure, done

**Pull request merged**